src/netconfig.cpp

```diff
@@ -379,7 +381,7 @@ static const Command ifconfigCommands[] = {
     {"reset", nullptr, "Reset configuration to factory defaults", cmdReset},
     {"mac", "{INTERFACE} MAC", "Set MAC address", cmdMac},
     {"hostname", "NAME", "Set host name", cmdHostname},
-    {"gateway", "IP", "Set default gateway", cmdGateway},
+    {"gateway", "IP", "Set default gateway", cmdGateway, "NOTE: The system doesn't allow unreachable gateways. Please ensure that your gateway \nbelongs to the network defined by the IP address and the Netmask."},
```

Comment:
Moreover, in the code this line also looks weird.
Could you reformat it to fit into 80 characters long?
I know that other lines also don't fit, but this line is too long. ;)